phpBB/docs/events.md

```diff
@@ -1042,6 +1042,22 @@ memberlist_search_sorting_options_before
 * Since: 3.1.2-RC1
 * Purpose: Add information before the search sorting options field.
 
+memberlist_team_username_append
+===
+* Locations:
+    + styles/prosilver/template/memberlist_team.html
+    + styles/subsilver2/template/memberlist_team.html
+* Since: 3.1.11-RC1
+* Purpose: Add information after team user username
```

Comment:
Why not just `Append information to username of team member`. Pretty much same except for prepend should work for the other event.